tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -766,6 +766,36 @@ public function theUserListsTheContentOfAPersonalSpaceRootUsingTheWebDAvApi(
 		);
 	}
 
+	/**
+	 * @When /^user "([^"]*)" sends PATCH request to the space "([^"]*)" of user "([^"]*)" with data "([^"]*)"$/
+	 *
+	 * @param string $user
+	 * @param string $spaceName
+	 * @param string $owner
+	 * @param string $data
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function userSendsPatchRequestToTheSpaceOfUserWithData(string $user, string $spaceName, string $owner, string $data): void {
+		$space = $this->getSpaceByName($owner, $spaceName);
+		Assert::assertIsArray($space);
+		Assert::assertNotEmpty($spaceId = $space["id"]);
+		Assert::assertNotEmpty($space["root"]["webDavUrl"]);
```

Comment:
needed to check webDavUrl?

Reply:
no that's because I copy pasted, I'll remove it